src/components/register.js

```diff
@@ -17,17 +17,15 @@ export const Register = () => {
 
   const schema = yup.object({
     accountName: yup.string().required("Account name required"),
-    accountNumber: yup
-      .string()
-      .required("Account number required"),
+    accountNumber: yup.string().required("Account number required"),
     accountPin: yup
       .string()
       .min(4)
       .max(4)
```

Comment:
Since it should have a fixed length, use `length() `instead of `min()` and `max()`.